src/examples/blinkm/blinkm.go

```diff
@@ -11,27 +11,26 @@ func main() {
 	machine.I2C0.Configure(machine.I2CConfig{})
 
 	// Init BlinkM
-	machine.I2C0.WriteTo(0x09, []byte("o"))
+	machine.I2C0.WriteRegister(0x09, 'o', nil)
```

Comment:
I'd personally recommend having the demo check for error, unless it's worthless.

Reply:
No error is generated at the moment, so nothing to check for.